browserslist

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
is this needed for the sample to run?

Reply:
No, but it was added by Angular CLI so I kept it.